README.md

```diff
@@ -38,4 +38,30 @@ bundle exec jekyll serve
 
 ## Publish site
 
-TBD, but create the html with `bundle exec jekyll build` then check the content into `asf-site` branch
\ No newline at end of file
+This is currently a manual process. Basic steps are:
+
+#### Check out `main` and build site
```

Comment:
We have had success using the https://github.com/peaceiris/actions-gh-pages action to automate publishing static content into another branch for our docs. It's built for GitHub Pages, but setting `publish_branch` to `asf_site` and `publish_dir` to `_site` might be enough to make it work for this. Here's [our workflow] if you want to take a look.

[our workflow]: https://github.com/spiceai/docs/blob/trunk/.github/workflows/build_and_publish.yml

Reply:
Thank you @phillipleblanc  -- the arrow blog has an automated workflow too which I think we could take a friendly look at as well.

https://github.com/apache/arrow-site/blob/main/.github/workflows/deploy.yml

I will file a ticket to track the idea

Reply:
Filed https://github.com/apache/datafusion-site/issues/13